modules/ROOT/pages/dw-core-functions-groupby.adoc

```diff
@@ -3,35 +3,43 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<groupby1>>
-//* <<groupby2>>
-//* <<groupby3>>
-
 
 [[groupby1]]
-== groupBy(Array<T>, (item: T, index: Number) -> R): { &#40;R&#41;: Array<T> }
+== groupBy(Array<T>, (item: T, index: Number) -> R): { (R): Array<T> }
+
+Classifies the elements of a list (array) using the specified `criteria` function.
+
+The resulting object will have the grouping criteria as keys.
```

Comment:
The resulting object contains the grouping criteria as keys.

Reply:
Or maybe This function returns an object containing the grouping criteria as keys